source/ProSymbolEditor/ProSymbolEditorModule.cs

```diff
@@ -28,8 +28,7 @@ namespace ProSymbolEditor
     internal class ProSymbolEditorModule : Module
     {
         private static ProSymbolEditorModule _this = null;
-        public static bool _isEnabled = false;
-        public static string WorkspaceString = "militaryoverlay.gdb";
+        public SchemaDataModel DataModel { get; set; }
```

Comment:
I worry "SchemaDataModel"/"DataModel" might also be too generic and hard to understand in the code - the official name for this is the ["MilitaryOverlayInformationModel"](http://esriurl.com/MilitaryOverlayInformationModel)  so something like MilitaryOverlaySchema and MilitaryOverlayDataModel might be better